.github/workflows/release_notes.yml

```diff
@@ -42,7 +73,7 @@ jobs:
           HIGHLIGHTS: ${{ steps.highlights.outputs.highlights }}
 
       # Update the release PR body
-      - run: gh pr edit ${{ inputs.releasePr }} --body-file ${{ steps.release_notes.outputs.release_notes_path }}
+      - run: gh pr edit ${{ github.event_name == 'issue_comment' && github.event.issue.number || inputs.releasePr }} --body-file ${{ steps.release_notes.outputs.release_notes_path }}
```

Comment:
Yes. It may be formatted strangely but it is a ternary: 

```js 
if (github.event_name == 'issue_comment')
  return github.event.issue.number
else
  return inputs.releasePr
```